mne/minimum_norm/resolution_matrix.py

```diff
@@ -138,17 +391,60 @@ def get_cross_talk(resmat, src, idx, norm=False):
         Forward Operator.
     src : instance of SourceSpaces
         Source space used to compute resolution matrix.
-    idx : list of int
-        Vertex indices for which PSFs or CTFs to produce.
-    norm : bool
-        Whether to normalise to maximum across all CTFs (default: False).
+    idx : list of int | list of Label
+        Source for indices for which to compute PSFs or CTFs. If mode is None,
+        PSFs/CTFs will be returned for all indices. If mode is not None, the
+        corresponding summary measure will be computed across all PSFs/CTFs
+        available from idx.
+        Can be:
+
+        * list of integers : Compute PSFs/CTFs for all indices to source space
+          vertices specified in idx.
+        * list of Label : Compute PSFs/CTFs for source space vertices in
+          specified labels.
+
+    mode : None | 'mean' | 'max' | 'svd'
```

Comment:
DRY -- use `%(psf_ctf_mode)s` here for entries that are the same and add entries to `mne/utils/docs.py`

Reply:
That's my fun fact of the day. So I add docdict['psf_ctf_mode'] = """... with the docstring information to mne/utils/docs.py?

Reply:
Yes and then make sure any function that uses the `%(psf_ctf_mode)s` has either `@verbose` decorator (if it has a `verbose` kwarg) or `@fill_doc` (if it does not have `verbose` kwarg)